tenable/ot/schema/base.py

```diff
@@ -32,3 +33,13 @@ class AssetInfo:
 @dataclass
 class AssetInfoList(NodesList):
     nodes: List[AssetInfo]
+
+
+@dataclass
+class IP:
```

Comment:
Use this class also in tenable/ot/schema/events.py line:124 (dst_ip) and on line:139 (src_ip)

Reply:
Done